labman/gui/templates/library_pooling.html

```diff
@@ -193,21 +221,59 @@
           }
        },
     };
-   
-    var histlayout = {
+    var histogramLayout = {
       autosize: false,
       width: heatwidth,
       xaxis: {title: "Concentration"},
       yaxis: {title: "Number Samples"},
       title: 'DNA Concentration Distribution'
     };
-    var histdata = [trace, blank_trace];
+    var histogramData = [nonBlankData, blankData];
+
+    Plotly.plot($heatmap.attr('id'), heatmapData, heatmapLayout);
+    Plotly.plot($histogram.attr('id'), histogramData, histogramLayout);
+
+    // setup some interactivity to clip the values in the histogram
+    $(function() {
+      var text = 'Clip values greater than ';
+      $slider.slider({
+        min: minConcentration + 1,
+        max: 1000,
+        value: defaultClipping,
+        create: function() {
+          $sliderLabel.text(text + $(this).slider("value"));
+        },
+        slide: function(event, ui) {
+          $sliderLabel.text(text + ui.value);
+        },
+        change: function(event, ui) {
+          $sliderLabel.text(text + ui.value);
+          Plotly.update($heatmap.attr('id'), {zmax: parseFloat(ui.value)}, 0);
+        }
+      });
+      $resetButton.on('click', function() {
+        $slider.slider('value', defaultClipping);
+      });
+    });
 
-    
-    Plotly.plot(heatmap.attr('id'), data, layout);
-    Plotly.plot(histogram.attr('id'), histdata, histlayout);
   };
 
+  /**
+   * Determine an appropriate clipping value for the heatmap's maximum value.
+   *
+   * @param {Float} plateType String describing the plate type.
+   * @returns A maximum expected value for the colorscale of a heatmap.
+   *
+   */
+  function clippingForPlateType(plateType) {
+    if (plateType == '16S library prep') {
+      return 100;
+    }
+    else {
```

Comment:
maybe make this an `elif (plateType == 'shotgun library prep')` statement to explicitly address the possibility of other plate types?

These are pooling values, so correct defaults are 1000 for shotgun and 15 for 16S. 

IF using slider to clip, do we also need to set the slider range values?